modules/sildisco/src/Auth/Process/AddIdp2NameId.php

```diff
@@ -138,25 +141,25 @@ public function process(array &$state): void
         if (isset($state['metadataPath'])) {
             $metadataPath = $state['metadataPath'];
         }
-        $idpEntries = \Sil\SspUtils\Metadata::getIdpMetadataEntries($metadataPath);
+        $idpEntries = Metadata::getIdpMetadataEntries($metadataPath);
 
         $idpEntry = $idpEntries[$samlIDP];
 
         // The IDP metadata must have an IDPNamespace entry
         if (!isset($idpEntry[self::IDP_CODE_KEY])) {
-            throw new \SimpleSAML\Error\Exception(self::ERROR_PREFIX . "Missing required metadata entry: " .
+            throw new Error\Exception(self::ERROR_PREFIX . "Missing required metadata entry: " .
```

Comment:
Already commented about the Error\something above multiple times. Not going to comment on each one.